quotaclimat/data_ingestion/ingest_db/ingest_sitemap_in_db.py

```diff
@@ -11,12 +13,17 @@
 parser = ArgumentParser()
 parser.add_argument("-p", "--dbpwd")
 args = parser.parse_args()
-DB_PWD = args.dbpwd
+
+DB_PWD = os.environ.get('POSTGRES_PASSWORD', args.dbpwd)
 
 
 def run():
     for media, sitemap_conf in SITEMAP_CONFIG.items():
         try:
+            # is DB init with schemas ?
```

Comment:
No. the schemas were there https://github.com/dataforgoodfr/quotaclimat/tree/main/postgres/schemas